pl.yml

```
@@ -204,8 +204,9 @@ app:
       title: "Język"
       description: "Wybierz preferowany język wyświetlania dla aplikacji Windhawk."
       contribute: "<0>Współtwórz nowe tłumaczenie</0>."
-      credits: "Polskie tłumaczenie stworzył <0>Inkatail</0>."
+      credits: "Polskie tłumaczenie stworzył <0>Inkatail</0> oraz <0>CyprinusCarpio</0>."
       creditsLink: "https://github.com/Inkatail"
+      creditsLink: "https://github.com/CyprinusCarpio"
```

Thank you for the contribution! Unfortunately, this won't work, only one link is supported. I'll look into adding support for more links in the future. For now, you'll have to find a different solution. Perhaps:
```suggestion
      credits: "Polskie tłumaczenie stworzył <0>Inkatail</0> oraz CyprinusCarpio (https://github.com/CyprinusCarpio)."
      creditsLink: "https://github.com/Inkatail"
```